Testreglar/2.2.2/App/app-2.2.2a.json

```diff
@@ -0,0 +1,212 @@
+{
+	"namn": "App-2.2.2a Det er mogleg å pause, stoppe eller skjule innhald som automatisk bevegar seg, blinkar eller rullar 2023",
+	"id": "app-2.2.2a",
+	"testlabId": 503,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nn",
+	"kravTilSamsvar": "<p>For innhold som automatisk beveger seg, blinker, ruller, der handlingen ikke er avgjørende, gjelder følgende</p>\r\n<ul>\r\n<li>Det er en mekanisme for å pause, stoppe eller skjule innholdet eller laste nettsiden på nytt uten slikt innhold og\r\n<ul>\r\n<li>Mekanismen ligger enten på starten av nettsiden eller i direkte tilknytning til innholdet eller</li>\r\n</ul>\r\n</li>\r\n<li>Det finnes en dokumentert tastatursnarvei for å pause, stoppe eller skjule innholdet eller laste nettsiden på nytt uten slikt innhold og\r\n<ul>\r\n<li>Innholdet som er satt på pause blir gjenopptatt på samme sted eller i sanntid</li>\r\n</ul>\r\n</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva appside testar du?",
+			"ht": "<p>Beskriv appsida med få stikkord, eller vel i lista.</p>",
+			"type": "tekst",
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsida innhald som automatisk beveger seg, blinkar eller rullar?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjekk om det finst innhald på appsida som automatisk beveger seg, blinkar eller rullar.</p>\n<p>Eksempel på denne typen innhald er video, animasjonar, spel og rullande børsnoteringar.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appside har ikkje innhald som automatisk beveger seg, blinkar eller rullar."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Kva innhald testar du?",
+			"ht": "<p>Beskriv innhaldet som beveger seg, blinkar eller rullar, slik at det er mogleg å identifisere det i ettertid.</p>\n<p>Viss det gjeld fleire element, registrerer du eit og eit.</p>",
+			"type": "tekst",
+			"label": "Innhald som beveger seg, blinkar, rullar:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Varer bevegelsen, blinkinga eller rullinga i meir enn 5 sekund?",
+			"ht": "<p>Bruk stoppeklokke eller lignande for å ta tida. </p>\n<p><strong>Merk:</strong> Video eller anna innhald som beveger seg, blinkar eller rullar, men varar kortere enn 5 sekund, er likevel omfatta viss det går i loop.</p>",
```

Comment:
Bruk norsk ord for loop.

Reply:
Vi beholder den. Alle vet om begrep loop.
https://ordbokene.no/bm,nn/search?q=loop&scope=ei